cypress/integration/team/teams_spec.js

```diff
@@ -107,4 +107,45 @@ describe('Teams Suite', () => {
         // 9. Remove user from team
         cy.removeTeamMember(teamURL, user.firstName);
     });
+
+    it('TS14633 Leave all teams', () => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+
+        function leaveTeamUntilCant() {
```

Comment:
First of all, please rebase to master since we [separated Cypress commands](https://github.com/mattermost/mattermost-webapp/tree/master/cypress/support) into `api_commands` and `ui_commands`.

Could you break this `leaveTeamUntilCant` function into 2 Cypress commands: one could be `leaveTeam` to be added at `ui_commands` and the other is `apiGetTeams` at `api_commands` (see [Get teams API reference](https://api.mattermost.com/#tag/teams%2Fpaths%2F~1teams%2Fget) on parameters needed)